src/components/maybe-internal-link/types.ts

```diff
@@ -0,0 +1,5 @@
+type AnchorElementProps = JSX.IntrinsicElements['a']
+
+export interface MaybeInternalLinkProps extends AnchorElementProps {
+  href: AnchorElementProps['href']
```

Comment:
Interesting...curious why we need to redefine href here if it's already a part of the `AnchorElementProps` extension. Not a blocker, just curious.

Reply:
☝️ I had the same question. Good to know! 💯